### DISCUSSION TITLE
Upload/download node settings

### DISCUSSION THREAD

**Discussion by Haflinger73** [meshtastic/Discussions#143] [Android]
Hello everyone, unfortunately I found out today that it is no longer possible to upload/download node settings files in the Android app.
Until a few weeks ago, if a rak node lost its memory again after the firmware update, I could simply load the cfg file stored on the smartphone onto the node and all the settings were back.
How can I do this as easily as possible with the current app?
![up download settings](https://github.com/user-attachments/assets/cdcf0bc4-0696-4364-bf1f-1a142fe08573)

**Comment by rmeineke** [comment]
Not sure if it is an actual bug or something that is intended ... there is some weird behavior in my app (v2.5.12) regarding this particular feature as well. BUT . . . 

If I choose 'More details' on the active node, and then choose 'Remote Administration' at the very bottom, there is a backup/restore option just above the reboot/shutdown/reset buttons. Here it gets weird... If I then choose an option above the backup/restore, and then go back one screen the backup/restore option is gone. If I go back one more screen, then click 'Remote Administration' again the backup/restore option is there again. 

(I should make a screen recording of these steps  just for my own sanity. I have this same behavior on two different phones. I should probably figure out how to file a bug report as well!)

![Screenshot_2024-12-22-05-44-58](https://github.com/user-attachments/assets/17f2e48c-da5b-4227-8a5f-a3253fb49723)
![Screenshot_2024-12-22-05-45-06](https://github.com/user-attachments/assets/2712df7e-56f7-4018-8011-4f677c0f544c)
![Screenshot_2024-12-22-05-45-12](https://github.com/user-attachments/assets/c38019eb-193c-4bb8-a942-cb5ba8892271)
![Screenshot_2024-12-22-05-45-25](https://github.com/user-attachments/assets/05a196db-113d-4e26-a236-4ab8c4f829d0)

**Reply by rmeineke** [reply]
Video available on the Youtubes...

https://youtu.be/Mnm_jpbaxYQ

**Comment by rmeineke** [comment]
Had some time on my hands so I downloaded and installed a bunch of APKs.

Downloaded and installed the following:

v2.5.15 alpha ... same problematic behavior

v.2.5.12 beta  * * * Here is where the problematic behavior first shows up for me * * * 

2.5.11 alpha ... works fine ... import/export available

2.5.10 beta ... works fine

2.5.9 alpha ... works fine

2.5.8 alpha ... works fine ... import/export available

v.2.5.7.beta ... works fine ... import/export available

v.2.5.6 ... seems fine ... import/export always available under the remote administration tab

Unfortunately, my meager Python skills won't let me get much further than this investigation-wise. But I'll see if I can't get a bug report filed next.

**Comment by rmeineke** [comment]
https://github.com/meshtastic/Meshtastic-Android/issues/1501

**Comment by Haflinger73** [comment]
It's over (at least I think so). I have app version 2.5.15 (alpha/beta?) as soon as you change something in the settings, inport/export disappears. If you scroll straight down, the inport/export icons are there and work as usual.

**Comment by JimTheCowboy** [comment]
this has been resolved 
https://github.com/meshtastic/Meshtastic-Android/issues/1501

@andrekir
